bundle/CMakeLists.txt

```diff
@@ -74,7 +74,7 @@ option(DAWN_PYTHON_EXAMPLES "Build the python examples of generating HIR" OFF)
 
 if( DAWN_BUNDLE_PYTHON )
   if(NOT DEFINED PYTHON_EXECUTABLE)
-    find_package(PythonInterp 3.4 REQUIRED)
+    find_package(PythonInterp 3.5 REQUIRED)

```

Comment:
Do we really need Python >=3.5? Maybe it should be enough to do
```
find_package(PythonInterp 3 REQUIRED)
```

Reply:
https://github.com/MeteoSwiss-APN/dawn/blob/f49685c38a60370f91adbf64e548a4bfc90ec599/python/dawn/sir.py#L19

is a python 3.5 feature